.circleci/config.yml

```diff
@@ -12,7 +12,6 @@ jobs:
   build:
     # The resource_class feature allows configuring CPU and RAM resources for each job. Different resource classes are available for different executors. https://circleci.com/docs/2.0/configuration-reference/#resourceclass
     resource_class: large
-    parallelism: 10
     working_directory: ~/openreview-expertise
```

Comment:
I remember you explained why we had to remove the parallelism, but can you remind me? That way it will be documented too. Thanks!